packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -137,6 +149,41 @@ describe( 'Reusable blocks', () => {
 		expect( paragraphContent ).toMatch( 'Oh! Hello there!' );
 	} );
 
+	// Check for regressions of https://github.com/WordPress/gutenberg/issues/33072.
+	it( 'can be saved when modified inside of a published post', async () => {
+		await createReusableBlock(
+			'Guten Berg!',
+			'Alternative greeting block'
+		);
+
+		// Make sure the reusable block has loaded properly before attempting to publish the post.
+		await page.waitForSelector( 'p[aria-label="Paragraph block"]' );
+
+		await publishPost();
+
+		// Close publish panel.
```

Comment:
should this part of the publishPost utility?

Reply:
We have some tests that click the publish panel's "View Post" link right after publishing, so we'd have to modify those. If it's okay, I'd thus leave `publishPost` untouched for now, in order to get this PR in without further detours :)